client/src/routes/Temple/Portal.tsx

```diff
@@ -128,6 +128,7 @@ const Portal: React.FC = () => {
         onEnd={() => navigate('Temple', {templeId})}
         paused={!joiningTemple}
         source={{uri: introPortal.content.videoEnd?.source}}
+        poster={introPortal.content.videoEnd?.thumbnail}
```

Comment:
I beleive there is a `posterResizeMode` that needs to sync with `resizeMode` for it to scale properly :)

Reply:
Nice, added those